Source/WebCore/html/ColorInputType.cpp

```diff
@@ -246,8 +248,8 @@ void ColorInputType::attributeChanged(const QualifiedName& name)
     if (name == valueAttr) {
         updateColorSwatch();
 
-        if (CheckedPtr cache = element()->document().existingAXObjectCache())
-            cache->valueChanged(*element());
+        if (CheckedPtr cache = element()->protectedDocument()->existingAXObjectCache())
```

Comment:
Might as well take the free speedup and put protectedDocument in a local here.

Reply:
I've cached `element()` which is I think what you meant, since I'm not using `document` again in the method.